frontend/pages/postulations/index.tsx

```diff
@@ -0,0 +1,64 @@
+import axiosWithoutAuth from "@/axiosWithoutAuth";
+import ModuleCard from "@/components/cards/ModuleUnionCard";
+import Layout from "@/components/Layout";
+import PageHeader from "@/components/PageHeader";
+import Pagination from "@/components/Pagination";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import { GetServerSideProps } from "next";
+import Link from "next/link";
+import { FC } from "react";
+
+interface PostulationsProps {
+  postulationsData: any;
+}
+
+const Postulations: FC<PostulationsProps> = ({ postulationsData }: PostulationsProps) => {
+  const postulations = postulationsData["results"] || [];
+  const postulationCards = postulations.map((postulation) => (
+    <Grid item key={postulation.slug} xs={6} sm={4} md={3}>
+      <Link href={`/postulations/${postulation.slug}`}>
+        <a>
+          <ModuleCard module={postulation}>
+            <div dangerouslySetInnerHTML={{ __html: postulation.summary }} />
+          </ModuleCard>
+        </a>
+      </Link>
+    </Grid>
+  ));
+
+  return (
+    <Layout title={"Postulations"}>
+      <Container>
+        <PageHeader>Postulations</PageHeader>
+        <Pagination count={postulationsData["total_pages"]} />
+        <Grid container spacing={2}>
+          {postulationCards}
+        </Grid>
+        <Pagination count={postulationsData["total_pages"]} />
+      </Container>
+    </Layout>
+  );
+};
+
+export default Postulations;
+
+// https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  let postulationsData = {};
+
+  await axiosWithoutAuth
+    .get("http://django:8000/api/postulations/", { params: context.query })
```

Comment:
Does this need to be changed to graphql? We're referencing attributes above (e.g. `slug`) that are not in the `PostulationSerializer`.
https://github.com/ModularHistory/modularhistory/blob/01a44f9e0be0a64d8bb76c7ff4284d7d66f9a2ac/apps/postulations/serializers.py#L8-L19

Reply:
We can't use GraphQL for list pages yet, since our GraphQL API doesn't return the `total_pages` field used by the pagination component, so for now, I'll just update the serializer.